#!/bin/sh

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01383-L1_S41_L007_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01383-L1_S41_L007_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01383_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01384-L1_S42_L007_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01384-L1_S42_L007_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01384_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01385-L1_S43_L007_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01385-L1_S43_L007_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01385_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01386-L1_S1_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01386-L1_S1_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01386_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01387-L1_S2_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01387-L1_S2_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01387_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01388-L1_S3_L002_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01388-L1_S3_L002_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01388_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01389-L1_S4_L003_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01389-L1_S4_L003_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01389_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01390-L1_S5_L003_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01390-L1_S5_L003_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01390_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01391-L1_S6_L004_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01391-L1_S6_L004_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01391_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01392-L1_S7_L005_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01392-L1_S7_L005_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01392_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01393-L1_S8_L005_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01393-L1_S8_L005_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01393_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01394-L1_S9_L005_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01394-L1_S9_L005_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01394_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01395-L1_S10_L007_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01395-L1_S10_L007_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01395_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01396-L1_S11_L007_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01396-L1_S11_L007_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01396_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01397-L1_S12_L007_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01397-L1_S12_L007_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01397_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01398-L1_S1_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01398-L1_S1_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01398_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01399-L1_S2_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01399-L1_S2_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01399_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01400-L1_S3_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01400-L1_S3_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01400_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01401-L1_S1_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01401-L1_S1_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01401_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01402-L1_S2_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01402-L1_S2_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01402_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01403-L1_S3_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01403-L1_S3_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01403_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01404-L1_S7_L005_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01404-L1_S7_L005_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01404_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01405-L1_S8_L005_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01405-L1_S8_L005_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01405_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 4 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01406-L1_S9_L005_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01406-L1_S9_L005_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01406_Concat_1set_kaki.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01407-L1_S35_L006_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01407-L1_S35_L006_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01407-L1_S35_L006.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01408-L1_S36_L006_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01408-L1_S36_L006_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01408-L1_S36_L006.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01409-L1_S37_L006_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01409-L1_S37_L006_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01409-L1_S37_L006.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01410-L1_S38_L006_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01410-L1_S38_L006_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01410-L1_S38_L006.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01411-L1_S39_L006_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01411-L1_S39_L006_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01411-L1_S39_L006.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01412-L1_S40_L006_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01412-L1_S40_L006_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01412-L1_S40_L006.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01413-L1_S10_L007_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01413-L1_S10_L007_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01413-L1_S10_L007.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01414-L1_S11_L007_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01414-L1_S11_L007_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01414-L1_S11_L007.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01415-L1_S12_L007_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01415-L1_S12_L007_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01415-L1_S12_L007.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01416-L1_S1_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01416-L1_S1_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01416-L1_S1_L001.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01417-L1_S2_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01417-L1_S2_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01417-L1_S2_L001.bam

/share/data3/conda/miniconda3/bin/bowtie2 -p 32 -x /share/data3/sga94/Genomes/Kaki_SuperScaffolds/Kaki_v2_Concat/Bowtie_Indexed/Kaki1_v2.2_concat_index.fasta -1 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01418-L1_S3_L001_R1_001_paired_trimmed.fq -2 /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01418-L1_S3_L001_R2_001_paired_trimmed.fq --very-sensitive -X 600 -I 0 | /share/data3/conda/miniconda3/bin/samtools view -bS - > /share/data3/sga94/IKMB_Data/fastq/trimmed/trimmomatic/H01418-L1_S3_L001.bam

